relaton/serializers/bibxml/reference.py

```diff
@@ -120,7 +120,13 @@ def create_reference(item: BibliographicItem) -> Element:
     except ValueError:
         pass
     else:
-        ref.set('target', target)
+        if item.doctype == 'internet-draft':
+            ref.append(E.format(
+                type="TXT",
+                target=target
+            ))
+        else:
+            ref.set('target', target)
```

Comment:
I believe we are to do this for all IETF types, including Internet-Drafts, RFC and RFC subseries.

Reply:
Looks like RFC entries do not have a doctype https://github.com/ietf-tools/relaton-data-rfcs/search?q=doctype . @ronaldtse what is the most reliable way to determine if a Relaton bibliographic item is a RFC?